web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -28,7 +28,10 @@
                 data-ng-href="catalog.search#/{{(md.draft == 'y')?'metadraf':'metadata'}}/{{md.uuid}}"
                 title="{{md.resourceAbstract}}"
                 class="gn-break"
-                >{{md.resourceTitle}}</a
+                >
+                <i data-ng-show="mdResourceTitle.trim()=='' || md.resourceTitle===undefined">{{'missingTitle' | translate}}</i>
```

Comment:
Is `mdResourceTitle` correct? Or should be `md.resourceTitle`?

Reply:
Ah yes, well spotted, I have modified this in the last commit, including some weird newlines due to the HTML prettifier, I'm afraid.